### PR TITLE
Fixes #1861 Animation fix

### LINKED ISSUE #1861: Wrong Animation Showing & Tab gets closed when we hit back from Settings/Bookmarks/History
**Describe the bug**
**Case1:** Wrong Animation Showing when we hit back from Settings.
**Case2:** Tab gets closed when we hit back from setting page opened from tab manager
and last activity is also killed.

**Expected behavior**
**Case1**: No Animation should appear.
**Case2:** Tab should not be closed/Open tab manager as left before or Open Home Page

**Steps to reproduce the behavior:**
1. Open Tab Manager
2. Close tab manager OR Leave it Open
3. Open Settings page from options
4. Hit Back
5. See Error
Error can be observed when we do the step 3 with history or bookmarks page.

**Screenshots**

**Case1**
case1
(Wrong Animation Showing when we hit back from Settings.
Tabs Slide up animation is displayed which should not done so.)

**Case2**
case2
(Tab gets closed when we hit back from setting page opened from tab manager
and last activity is also killed.)

**Environment**
- Version of Kiwix Android : 3.2.1
- Device : Pixel 4
- OS version : Android 10

**Willing to work on this Issue:** Yes

### PR DESCRIPTION
**Pull request by 4shutosh**
Fixes #1861 
As per the issue

**Case1:** No Animation should appear.
**Case2:** Open Home Page

**Screenshots** 
ezgif com-video-to-gif (4)

### CODE REVIEW

File: core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java [hunk 1]
```diff
@@ -556,10 +556,13 @@ protected void hideTabSwitcher() {
       setDrawerLockMode(DrawerLayout.LOCK_MODE_UNLOCKED);
       closeAllTabsButton.setImageDrawable(
         ContextCompat.getDrawable(this, R.drawable.ic_close_black_24dp));
-      startAnimation(tabSwitcherRoot, R.anim.slide_up);
-      tabSwitcherRoot.setVisibility(View.GONE);
-      progressBar.setVisibility(View.VISIBLE);
-      contentFrame.setVisibility(View.VISIBLE);
+      if(tabSwitcherRoot.getVisibility()==View.VISIBLE){
```
Comment (srv-twry) [inline core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java]: space needed before and after `==`

**Comment by 4shutosh** [comment]
I have made this pr for the issue #1861 
Please review @abdulwd @mhutti1 @kelson42 !

File: core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java [hunk 2]
```diff
@@ -556,10 +556,13 @@ protected void hideTabSwitcher() {
       setDrawerLockMode(DrawerLayout.LOCK_MODE_UNLOCKED);
       closeAllTabsButton.setImageDrawable(
         ContextCompat.getDrawable(this, R.drawable.ic_close_black_24dp));
-      startAnimation(tabSwitcherRoot, R.anim.slide_up);
-      tabSwitcherRoot.setVisibility(View.GONE);
-      progressBar.setVisibility(View.VISIBLE);
-      contentFrame.setVisibility(View.VISIBLE);
+      if(tabSwitcherRoot.getVisibility() == View.VISIBLE){
```
Comment (abdulwd) [inline core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java]: The code is not properly formatted.

Comment (4shutosh) [reply core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java]: Done!

File: core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java [hunk 3]
```diff
@@ -556,10 +556,13 @@ protected void hideTabSwitcher() {
       setDrawerLockMode(DrawerLayout.LOCK_MODE_UNLOCKED);
       closeAllTabsButton.setImageDrawable(
         ContextCompat.getDrawable(this, R.drawable.ic_close_black_24dp));
-      startAnimation(tabSwitcherRoot, R.anim.slide_up);
-      tabSwitcherRoot.setVisibility(View.GONE);
-      progressBar.setVisibility(View.VISIBLE);
-      contentFrame.setVisibility(View.VISIBLE);
+      if (tabSwitcherRoot.getVisibility() == View.VISIBLE) {
+          tabSwitcherRoot.setVisibility(View.GONE);
```
Comment (mhutti1) [inline core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java]: We use 2 space indents not 4.

Comment (4shutosh) [reply core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java]: Appologies.
This won't happend again !

**Pull request merged**